apps/website/src/content/docs/flex.mdx

```diff
@@ -1,19 +1,43 @@
 ---
 title: Flex
-description: A utility component to work with CSS flexbox and iTwinUI design tokens.
+description: A utility component that makes it easier to work with CSS flexbox and iTwinUI design tokens.
 thumbnail: #TODO
 group: utilities
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='flex--basic' />
+## Usage
+
+Wrap `Flex` around the components that should be the children of the flex container. Set the gap using the `gap` prop with a value of an iTwinUI size token (e.g. `x`, `xs`) or any valid css value (e.g. "1rem"). The default `gap` is `xs`.
 
 <LiveExample src='Flex.main.jsx'>
   <AllExamples.FlexMainExample client:load />
 </LiveExample>
 
-A utility component that makes it easier to work with CSS flexbox and iTwinUI design tokens. Supports all flex properties. Can be used with or without `Flex.Item` and `Flex.Spacer` subcomponents.
+The `Flex` container offers props that are equivalent to those of the CSS flexbox, such as `alignItem`, `justifyContent`, `flexDirection`, and `flexWrap`.
```

Comment:
Typo fix + mention of default value (because it's different from CSS flexbox default)

```suggestion
The `Flex` container offers props that are equivalent to those of the CSS flexbox, such as `alignItems` (defaults to `"center"`), `justifyContent`, `flexDirection`, and `flexWrap`.
```

Reply:
I have made this change.